media-libs/opensubdiv/opensubdiv-3.3.2.recipe

```diff
@@ -83,8 +83,8 @@ BUILD()
 		GLEW_SUBDIR="$secondaryArchSuffix/"
 	fi
 
-	export GLEW_LOCATION="$portPackageLinksDir/glew${secondaryArchSuffix}_devel/develop"
-	export GLEW_INC_LOCATION="$portPackageLinksDir/glew${secondaryArchSuffix}_devel/develop/headers$UNITTEST_SUBDIR"
+	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/develop/"
+	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/develop/headers/$GLEW_SUBDIR"
```

Comment:
Suggestion:
```diff
-	local GLEW_SUBDIR=
-	if [ -n "$secondaryArchSuffix" ]; then
-		GLEW_SUBDIR="$secondaryArchSuffix/"
-	fi
-
-	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/develop/"
+	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/$relativeDevelopDir/"
-	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/develop/headers/$GLEW_SUBDIR"
+	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/$relativeIncludeDir/"
```
Warning: I did not test this.

Same suggestion for version `3.3.3`.

BTW, unlike `3.3.3`, `3.3.2` is missing a `SOURCE_FILENAME`, so you might wish to add one:
```diff
 CHECKSUM_SHA256="d11a0f21bcf21156f9071ea49050ec6675b77776b9dba33b8e30b072d82a8b05"
+SOURCE_FILENAME="opensubdiv-$portVersion.tar.gz"
 SOURCE_DIR="OpenSubdiv-${portVersion//\./_}"
```